server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -259,34 +269,103 @@ private TrTag renderTIRow(AccountModel ti) {
+                .withClass("usa-card__container")
+                .with(
+                    div()
+                        .withClasses("usa-card__header", "flex justify-between")
+                        .with(
+                            div(
+                                    div(
+                                            renderSubHeader(account.getApplicantName())
+                                                .withClass("usa-card__heading"),
+                                            u(renderUpdateClientLink(account.id)).withClass("ml-2"))
+                                        .withClass("flex"),
+                                    renderCardDateOfBirth(account))
+                                .withClass("flex-col"),
+                            renderIndexPageLink(account)),
+                    div()
+                        .withClasses("usa-card__body", "flex")
+                        .with(
+                            renderCardContactInfo(account).withClasses("w-2/5"),
+                            renderCardApplications(account).withClasses("ml-10 w-2/5"),
+                            renderCardNotes(account.getTiNote()).withClasses("ml-10 w-3/5"))));
+  }
+
+  private DivTag renderCardContactInfo(AccountModel account) {
+    Optional<ApplicantModel> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return div();
+    }
+    ApplicantData applicantData = newestApplicant.get().getApplicantData();
+    Optional<String> maybePhoneNumber = applicantData.getPhoneNumber();
+    String email = account.getEmailAddress();
+
+    return div(
+        label("Contact information").withFor("card_contact_info").withClass("whitespace-nowrap"),
+        div()
+            .condWith(
+                maybePhoneNumber.isPresent(),
+                div(
+                        Icons.svg(Icons.PHONE).withClasses("h-3", "w-3", "mr-1"),
+                        p(formatPhone(maybePhoneNumber.orElse(""))))
+                    .withClass("flex items-center"))
+            .condWith(
+                email != null && !email.isEmpty(),
+                div(Icons.svg(Icons.EMAIL).withClasses("h-3", "w-3", "mr-1"), p(email))
+                    .withClass("flex items-center"))
+            .withClass("text-xs")
+            .withId("card_contact_info"));
+  }
+
+  private String formatPhone(String phone) {
+    return phone.replaceFirst("(\\d{3})(\\d{3})(\\d+)", "($1) $2-$3");
+  }
+
+  private DivTag renderCardApplications(AccountModel account) {
+    Optional<ApplicantModel> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return div();
+    }
+    int applicationCount = newestApplicant.get().getApplications().size();
+
+    String programs =
+        newestApplicant.get().getApplications().stream()
+            .map(
+                application ->
+                    programRepository.getProgramDefinition(application.getProgram()).adminName())
```

Comment:
Should this be the program's admin name or the program's localized name? We have a bug in Reporting where we had to change to programLocalizedName as Admin name wasn't clear. https://github.com/civiform/civiform/pull/5797

Reply:
Ah, good point.  I see now in the mocks that the localized name is being used.  Will change to use the localized name.  Thanks for catching that.